src/eureka/S5_lightcurve_fitting/differentiable_models/QuasiLambertianPhaseCurve.py

```diff
@@ -90,7 +90,7 @@ def eval(self, eval=True, channel=None, pid=None, **kwargs):
 
                 if (eval and pl_params.quasi_gamma == 0):
                     # Don't waste time running the following code
-                    phaseVars = np.ma.ones_like(time)
+                    phaseVars = np.ma.ones(time.shape)
```

Comment:
Why change from `np.ma.ones_like(time)` to `np.ma.ones(time.shape)`?  They're effectively the same, right?

Reply:
Because `np.ma.ones_like` doesn't exist, frustratingly...